aesara/tensor/nnet/basic.py

```diff
@@ -14,6 +14,7 @@
 """
```

Comment:
Shouldn't we rename this from `nnet` to something else? Or just move it down into the regular namespace.